vars/example-vars-staging.yml

```diff
@@ -5,7 +5,7 @@
 - app_user_gid:  123000
 - app_user_uid:  987000
 - deploy_root: /hydra-dev
-- deploy_users:
+- db_deploy_users:
```

Comment:
We shouldn't have to specify the prefixed (namespaced) variable names directly.  There's a ruby script that should take care of expanding `deploy_users` the correctly, and generating a vars file that has all the variables.